application/src/main/kotlin/com/studentcenter/weave/application/service/application/UserSocialLoginApplicationService.kt

```diff
@@ -0,0 +1,40 @@
+package com.studentcenter.weave.application.service.application
+
+import com.studentcenter.weave.application.port.inbound.UserSocialLoginUseCase
+import com.studentcenter.weave.application.service.UserTokenService
+import com.studentcenter.weave.application.service.domain.UserAuthInfoDomainService
+import com.studentcenter.weave.application.service.domain.UserDomainService
+import com.studentcenter.weave.application.vo.UserTokenClaims
+import com.studentcenter.weave.domain.entity.User
+import com.studentcenter.weave.domain.entity.UserAuthInfo
+import org.springframework.stereotype.Service
+
+@Service
+class UserSocialLoginApplicationService(
+    private val userTokenService: UserTokenService,
+    private val userDomainService: UserDomainService,
+    private val userAuthInfoDomainService: UserAuthInfoDomainService,
+) : UserSocialLoginUseCase {
+
+    override fun invoke(command: UserSocialLoginUseCase.Command): UserSocialLoginUseCase.Result {
+        val idTokenClaims: UserTokenClaims.IdToken = userTokenService.resolveIdToken(
+            idToken = command.idToken,
+            provider = command.socialLoginProvider,
+        )
+
+        val userAuthInfo: UserAuthInfo = userAuthInfoDomainService.findByEmail(email = idTokenClaims.email)
```

Comment:
p5) 단일 인자에 메서드명에 표기되고 있어서?!
```suggestion
        val userAuthInfo: UserAuthInfo = userAuthInfoDomainService.findByEmail(idTokenClaims.email)
```